06_leapYears/leapYears.js

```diff
@@ -1,5 +1,12 @@
-const leapYears = function() {
-
+const leapYears = function (largeYear) {
+  if (largeYear % 4 == 0 && largeYear % 100 != 0) {
+    return true;
+  } else {
+    if (largeYear % 400 == 0) {
+      return true;
+    }
```

Comment:
Would it be possible to somehow don't use so much nesting?
It feels like it should be possible to use only one if, something like that
```js
if (condition1 || condition2) return true
```